python/ParcelDelineation/README.md

```diff
@@ -0,0 +1,25 @@
+# Parcel delineation
+This is an [openEO](https://openeo.org/) example for delineating agricultural parcels based on a neural network, using Sentinel-2 input data.
+
+The example focuses on the inference step, using a trained model. It demonstrates data loading and preprocessing, 
+inference, and finally producing vector data as a result.
+
+The example serves as a technology demonstration for openEO, and is not intended for use in another context. 
+Please contact the authors in case you're interested in a field detector!
+
+## Authors
+
+- Kristof Van Tricht
+- Jeroen Dries
+- Victor Verhaert
+
+Tuning by:
+- Kasper Bonte
+- Bart Driessen
+
+[VITO Remote Sensing](https://remotesensing.vito.be)
+
+## Running the example
+
+You will require an openEO account to run the example. The [openEO platform](https://openeo.cloud/) offers various options for this.
```

Comment:
best to remove this sentence about accounts, as this is a platform independent example. Info about how to get accounts is available elsewhere.

Reply:
I have replaced it with a reference to the openEO hub for finding providers.